src/helpers/mail.ts

```diff
@@ -0,0 +1,15 @@
+import mail from '../config/mailer'
+
+export interface Payload {
+  from: string;
+  to: string;
+  subject: string;
+  text: string;
+  html?: string;
+}
+
```

Comment:
Tambahan dikit pakai method verify apakah smtp email host ready or disconnected ref https://github.com/jabardigitalservice/pikobar-pelaporan-backend/blob/master/helpers/email/index.js

Reply:
kalau disconnected trus di apain mang

Reply:
up todo next pr and discussiin